pmm-app/src/shared/components/Form/NumericInput/NumericInputField.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { Field } from '@grafana/ui';
```

Comment:
I don't think Grafana's Field component is the way to go forward. Better use FinalForm's analogous one. Much more stable and tested.

Reply:
Done 👍